tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -85,7 +85,7 @@ bool GetEnabledDefaultValue()
                                                || val.Equals(UserTrackingExtendedMode, StringComparison.OrdinalIgnoreCase))
                                          .ToLowerInvariant();
 
-            ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiExperimentalSecurityEnabled)
+            ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiSecurityEnabled)
```

Comment:
If you want to be super safe, you could add `DD_EXPERIMENTAL_API_SECURITY_ENABLED` as a fallback. I don't think this is _required_ as you've checked metabase, but the "safer" thing to do would be to include the fallback for now, and then remove it in v3

```suggestion
            ApiSecurityEnabled = config.WithKeys(ConfigurationKeys.AppSec.ApiSecurityEnabled, "DD_EXPERIMENTAL_API_SECURITY_ENABLED")
```